includes/razorpay-webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
```

Comment:
Found a couple of nice suggestions on how to do webhooks in wordpress cleanly:

- https://wordpress.stackexchange.com/questions/178090/best-way-for-plugin-to-accept-posts
- https://wordpress.stackexchange.com/questions/126572/trying-to-avoid-including-wp-load-php/126582#126582